parsec/core/config.py

```diff
@@ -54,8 +54,8 @@ class CoreConfig:
     data_base_dir: Path
     cache_base_dir: Path
     mountpoint_base_dir: Path
+    pattern_filter_path: Optional[Path] = None  # None means: use the default filter path
```

Comment:
Wouldn't it be simpler for the core to directly store here the pattern object (and let the configuration loader do the dirty work of retrieving/building the pattern) ?

Reply:
Does it really make any difference? I wanted to avoid putting too much logic in the configuration module

Reply:
I guess we can rework this later if it's too much of a hassle... especially given this PR is far to long for it own good ;-)

In theory, this could be useful to provide a custom filter in test without having to create a file